templates/CommentReplies.ss

```diff
@@ -1,10 +1,12 @@
 <% if $RepliesEnabled %>
 	<div class="comment-replies-container">
-		
 		<div class="comment-reply-form-holder">
-			$ReplyForm
+            <% if $JavaScriptEnabled %>$ReplyForm<% else %>
+            <%-- non ajax, only show if replyTo is set --%>
+            <% if $ShowReplyToForm %>$ReplyForm<% end_if %>
```

Comment:
What if both $JavascriptEnabled is true and $ShowReplyToForm is true? Won't $ReplyForm end up being assigned twice?